<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReserveTicket</class>
 <widget class="QDialog" name="ReserveTicket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>479</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Seat_A1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>A1</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_A2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>A2</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_A3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>A3</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_B1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>B1</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_B2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>B2</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_B3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>B3</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_C1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>C1</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_C2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>C2</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_C3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>C3</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_D1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>D1</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_D2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>D2</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_D3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>D3</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_E1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>E1</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_E2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>E2</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_E3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>E3</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_F1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>F1</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_F2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>F2</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_F3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>F3</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_G1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>G1</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_G2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>G2</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_G3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>G3</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_H1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>H1</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_H2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>H2</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_H3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>H3</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_I1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>I1</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_J1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>J1</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_I2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>I2</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_J2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>410</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>J2</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_I3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>370</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>I3</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Seat_J3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>410</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>J3</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">seatButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Passenger Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_passengerName">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>110</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_gender">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>160</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Male</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Female</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_mobileNo">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>160</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>140</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Mobile No.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>140</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Gender</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>230</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Selected Seats:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_selectedSeats">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>250</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_confirm">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>310</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_reset">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>310</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="seatButtonGroup"/>
 </buttongroups>
</ui>
